topics/racket-languages/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: racketlanguages
+created_by: PLT Inc.
+display_name: Racket Languages
+github_url: https://github.com/racket
+logo: racket.png
+related: drracket, racket
+released: 1995
+short_description: The 'racket-languages' topic is for programming languages implemented on the Racket platform.
```

Comment:
I like this! But maybe we can shorten it further to be punchier?

```suggestion
short_description: Programming languages implemented on the Racket platform.
```

Reply:
There is already a ‘racket’ topic.
The intention of the Racket-languages was to tag languages implemented with
the racket tool chain.

I think I should withdraw this topic. A collection would make more sense,
but was rejected on the grounds that racket isn’t well known enough.

Tvm

S.